galaxy_importer/utils/markup.py

```diff
@@ -72,6 +71,8 @@ def get_html(doc_file):
     """
     if doc_file.mimetype == "text/markdown":
         return _render_from_markdown(doc_file)
+    elif doc_file.mimetype == "text/x-rst":
+        return _render_from_restructured_text(doc_file)
```

Comment:
probably need to pull out the html sanitization from _render_from_markdown so it can be called on the unsafe html coming from the rst method